Makefile

```diff
@@ -137,65 +130,30 @@ lint: ##@tests Run meta linter on code
 
-mock-install:
+mock-install: ##@other Install mocking tools
 	go get -u github.com/golang/mock/mockgen
 
-mock: mock-install ##@other Regenerate mocks
+mock: ##@other Regenerate mocks
 	mockgen -source=geth/common/types.go -destination=geth/common/types_mock.go -package=common
 
-test: ##@tests Run tests
-	@build/env.sh echo "mode: set" > coverage-all.out
-	build/env.sh go test -coverprofile=coverage.out -covermode=set ./geth/api
-	@build/env.sh tail -n +2 coverage.out >> coverage-all.out
-	build/env.sh go test -coverprofile=coverage.out -covermode=set ./geth/common
-	@build/env.sh tail -n +2 coverage.out >> coverage-all.out
-	build/env.sh go test -coverprofile=coverage.out -covermode=set ./geth/jail
-	@build/env.sh tail -n +2 coverage.out >> coverage-all.out
-	build/env.sh go test -coverprofile=coverage.out -covermode=set ./geth/node
-	@build/env.sh tail -n +2 coverage.out >> coverage-all.out
-	build/env.sh go test -coverprofile=coverage.out -covermode=set ./geth/params
-	@build/env.sh tail -n +2 coverage.out >> coverage-all.out
-	build/env.sh go test -coverprofile=coverage.out -covermode=set ./extkeys
-	@build/env.sh tail -n +2 coverage.out >> coverage-all.out
-	build/env.sh go test -coverprofile=coverage.out -covermode=set ./cmd/statusd
-	@build/env.sh tail -n +2 coverage.out >> coverage-all.out
-	@build/env.sh go tool cover -html=coverage-all.out -o coverage.html
-	@build/env.sh go tool cover -func=coverage-all.out
-
-test-api:
-	build/env.sh go test -v -coverprofile=coverage.out  -coverpkg=./geth/node ./geth/api
-	@build/env.sh go tool cover -html=coverage.out -o coverage.html
-	@build/env.sh go tool cover -func=coverage.out
-
-test-common:
-	build/env.sh go test -v -coverprofile=coverage.out ./geth/common
-	@build/env.sh go tool cover -html=coverage.out -o coverage.html
-	@build/env.sh go tool cover -func=coverage.out
-
-test-jail:
-	build/env.sh go test -v -coverprofile=coverage.out ./geth/jail
-	@build/env.sh go tool cover -html=coverage.out -o coverage.html
-	@build/env.sh go tool cover -func=coverage.out
-
-test-node:
-	build/env.sh go test -v -coverprofile=coverage.out ./geth/node
-	@build/env.sh go tool cover -html=coverage.out -o coverage.html
-	@build/env.sh go tool cover -func=coverage.out
-
-test-params:
-	build/env.sh go test -v -coverprofile=coverage.out ./geth/params
-	@build/env.sh go tool cover -html=coverage.out -o coverage.html
-	@build/env.sh go tool cover -func=coverage.out
-
-test-extkeys:
-	build/env.sh go test -v -coverprofile=coverage.out ./extkeys
-	@build/env.sh go tool cover -html=coverage.out -o coverage.html
-	@build/env.sh go tool cover -func=coverage.out
-
-test-cmd:
-	build/env.sh go test -v -coverprofile=coverage.out ./cmd/statusd
-	@build/env.sh go tool cover -html=coverage.out -o coverage.html
-	@build/env.sh go tool cover -func=coverage.out
+test-unit: ##@tests Run unit tests
```

Comment:
Can we also add here `test` metatarget for local test runs? (probably should be the same as `ci`)

Reply:
Yeah, totally. With better unit tests coverage I think `test: test-unit` would be enough but I guess currently it should be as you said.